src/style.css

```diff
@@ -0,0 +1,100 @@
+* {
+  --background-color-field: #fff;
+  --text-color-field: #3d4e61;
+  --border-color-field: #e1e7ed;
+}
+
+@font-face {
+  font-family: Avenir-normal;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: Avenir-bold;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+form {
+  margin-top: 20px;
+}
+
+.area {
+  width: 100%;
+  border: 1px solid var(--border-color-field);
+  border-radius: 4px;
+  font-weight: 300;
+  color: var(--text-color-field);
+  background-color: var(--background-color-field);
+  font-family: Avenir-normal, serif;
+}
+
+.bigField {
+  height: 70px;
+  font-size: 16px;
+}
+
+.bigField:hover {
+  box-shadow: 0 4px 4px 0 rgba(0, 0, 0, 0.25);
+}
+
+.bigField:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  font-size: 16px;
+  line-height: 22px;
+  color: var(--text-color-field);
+  font-family: Avenir-bold, serif;
+  font-style: normal;
+}
+
+.smallField {
+  height: 42px;
+  font-size: 14px;
+}
+
+.smallField:hover {
+  box-shadow: 0 3px 8px 0 rgba(61, 78, 97, 0.2);
+}
+
+.smallField:focus {
```

Comment:
move repeated styles to common selector (.area)
And I'd recommend to change naming
Use
.input
.input--small
.input--big